cypress/support/dictionary/permissions.js

```diff
@@ -92,7 +92,6 @@ export default {
   uiUsersDeclareItemLost: { internal: 'ui-users.loans.declare-item-lost', gui: 'Users: User loans declare lost' },
   usersLoansRenewThroughOverride: { internal: 'ui-users.loans.renew-override', gui: 'Users: User loans renew through override' },
   uiUsersEditProfile: { internal: 'ui-users.edit', gui: 'Users: Can edit user profile' },
-  uiUsersViewProfile: { internal: 'ui-users.view', gui: 'Users: Can view user profile' },
```

Comment:
Why do you remove this permission? Other test reuse this permission, so another test will fail after this removal of the permission

Reply:
Please update this permission in all autotests where this permission is used.

Reply:
this permission is a duplicate of "uiUsersView: { internal: 'ui-users.view', gui: 'Users: Can view user profile' }". There was only one occurence in "cypress/e2e/bulk-edit/permissions/bulk-edit-user-permissions.cy.js" and it was replaced as you can see